### PR TITLE
Added automatic seeding of all available seeds.

### PR DESCRIPTION
**Pull request by nenti**

### CODE REVIEW

**Comment by AshanFernando** [comment]
@nenti Thanks for the PR, can you also update the Readme for this?

File: index.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ class ServerlessDynamodbLocal {
     get seedSources() {
         const config = this.service.custom.dynamodb;
         const seedConfig = _.get(config, "seed", {});
-        const seed = this.options.seed;
+        const seed = this.options.seed ? this.options.seed : Object.keys(seedConfig).join(',');
```
Comment (mattfysh) [inline index.js]: @nenti - this is awesome, in JS you could rep this as

```javascript
const seed = this.options.seed || Object.keys(seedConfig).join(',');
```

**Comment by mattfysh** [comment]
+1 would love to see this on npm

**Review by rehrumesh — APPROVED** [review]

**Comment by rehrumesh** [comment]
@nenti would love to merge this soon after the readme update.

**Comment by nenti** [comment]
@rehrumesh I added Readme changes. Aswell as some tweaks to the auto seeding. You can now use --seed command to overwrite your configuration. And with the old implementation using --seed without any seeds configured, the plugin crashed. So I changed the seed handler for more robustness. 

In the reedme I added an example seed because this is something that I had to figure out and would have loved to be available there. Although I think we could label it to make sure the user understands what it is. I'm not a pro in markdown.

**Review by rehrumesh — APPROVED** [review]

**Comment by rehrumesh** [comment]
@nenti Thanks for the awesomeness. 🥇

**Comment by amitaymolko** [comment]
It seems that this change hasn't been released yet.
When will this be available on npm?

**Comment by rehrumesh** [comment]
@amitaymolko @mattfysh @nenti NPM is updated with new release :)

**Pull request merged**